bar-trender-front-end/src/components/FreePieChart.js

```diff
@@ -1,42 +1,91 @@
-import React, {Component} from 'react';
+import React, { useEffect, useState, Component} from 'react';
 import {Bar, Line, Pie} from 'react-chartjs-2';
 
-class BeerPieChart extends Component{
-  constructor(props){
-    super(props);
-    this.state = 
-      this.props.data;
+
+  function BeerPieChart(props) {
     
-  }
+    const [modal1, setModal1] = React.useState(false);
+    const [modal2, setModal2] = React.useState(false);
+    const { element } = props;
 
-  static defaultProps = {
-    displayTitle:true,
-    displayLegend: true,
-    legendPosition:'right',
-    location:'City'
-  }
 
-  render(){
+    
+    const [appState, setAppState] = useState({
+      stats: {},
+    });
+    
+
+ 
+  
+     useEffect(() => {
+      var token = sessionStorage.getItem("token");
+      console.log(token,'token');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7173304)
```suggestion
      console.log(token,"token");
```